pkgs/applications/science/biology/dcm2niix/default.nix

```diff
@@ -17,7 +24,22 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ cmake git ];
-  buildInputs = [ libyamlcpp ];
+  buildInputs = with lib;
```

Comment:
I don't think we need with here. lib is so short and nice that we can just write it out.